dotnet restore example with path didn't work

[Enter feedback here]
I have tried the following command from 1 level up.
dotnet restore ~/projects/app1/app1.csproj

It kept on complaining the  error MSB1009: Project file does not exist.

But when I tried the following
dotnet restore ./projects/app1/

it was able to restore.

Host (useful for support):
  Version: 3.1.4
  Commit:  0c2e69caa6

Please let me know if you require more details.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0b256dc4-c4ad-c115-f4e2-40310bc3505a
* Version Independent ID: 0b993779-65a2-6092-4047-2fc2ec16170b
* Content: [dotnet restore command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore)
* Content Source: [docs/core/tools/dotnet-restore.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-restore.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

--- a/docs/core/tools/dotnet-restore.md
+++ b/docs/core/tools/dotnet-restore.md
@@ -155,10 +155,10 @@ There are three specific settings that `dotnet restore` ignores:
   dotnet restore
   ```
 
-- Restore dependencies and tools for the `app1` project found in the   given path:
+- Restore dependencies and tools for the `app1` project found in the given path:
 
   ```dotnetcli
-  dotnet restore ~/projects/app1/app1.csproj
+  dotnet restore ./projects/app1/app1.csproj
   ```
 
 - Restore the dependencies and tools for the project in the current   directory using the file path provided as the source:

